worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +53,31 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	std::string inputWord;
+	bool winGame = false;
+
+	std::cout << "The password is " << secret << "\n";
+
+	for (int numLives = 5; numLives > 0; numLives = numLives - 1) {
```

Comment:
Could use `numLives--` here. Also should use the constant `wordLength` instead of the literal `5`.